### PR TITLE
docs: Add documentation how to delete a T4C project

### PR DESCRIPTION
**Pull request by MoritzWeber0**

### CODE REVIEW

File: docs/user/docs/tools/capella/teamforcapella/project-management/project-management.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+# TeamForCapella project management
```
Comment (Paula-Kli) [inline docs/user/docs/tools/capella/teamforcapella/project-management/project-management.md:6]: I was wondering whether we want to have a look at using [title case](https://en.wikipedia.org/wiki/Title_case#Chicago_Manual_of_Style) throughout our docs or whether that is maybe just a minor enhancement

File: docs/user/docs/tools/capella/teamforcapella/project-management/project-management.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+1. Open a persistent Capella session and connect to it.
+1. Enable the TeamForCapella administration capability: <br /> `Window` >
+   `Preferences` > `General` > `Capabilities`:
+   ![Activate TeamForCapella administration capabilities](./activate-t4c-administration-guide.png)
+   Then close the dialog.
+1. Open the Capella search and search for `CDO Sessions (CDO)` and open it.
+1. Add a new CDO connection:
+   ![Add a new CDO connection](./add-new-cdo-session.png).
+1. Enter the connection string for the TeamForCapella server. There two ways to
+   find it out. Make sure that the host is prefixed with `tcp://`.
```
Comment (Paula-Kli) [inline docs/user/docs/tools/capella/teamforcapella/project-management/project-management.md:37]:
```suggestion
1. Enter the connection string for the TeamForCapella server. There two ways to
   find the correct one. Make sure that the host is prefixed with `tcp://`.
```

**Review by Paula-Kli — APPROVED** [review]

File: docs/user/docs/tools/capella/teamforcapella/project-management/project-management.md (hunk 1 above)
Comment (MoritzWeber0) [reply docs/user/docs/tools/capella/teamforcapella/project-management/project-management.md:6]: As this is not consistently used currently - let's address it in another PR. We agreed on using the "Chicago Manual of Style" for all titles, but should also apply to all existing titles.

Comment (MoritzWeber0) [reply docs/user/docs/tools/capella/teamforcapella/project-management/project-management.md]: EDIT: Changed the one specific occurrence.

**Pull request merged**